tests/Webapi/Dom/Webapi__Dom__MutationObserver__test.res

```diff
@@ -0,0 +1,31 @@
+open Webapi.Dom
+open MutationObserver
+
+let el = document->Document.createElement("strong")
+let observer: t = MutationObserver.make((_records, _observer) => ())
+
+let _ = observer->MutationObserver.observe(el, {
+  "subtree": true,
+  "childList": true,
+  "attributes": true,
+  "attributeOldValue": true,
+  "characterData": true,
+  "characterDataOldValue": true,
+})
+
+let _ = observer->MutationObserver.disconnect
+let records: array<Dom.mutationRecord> = observer->MutationObserver.takeRecords
+let aRecord = records->Js.Array2.unsafe_get(0)
+
+open MutationRecord
+
+let recordType: string = aRecord->type_
+let recordTarget: Dom.node = aRecord->target
+let recordAddedNodes: Dom.nodeList = aRecord->addedNodes
+let recordRemovedNodes: Dom.nodeList = aRecord->removedNodes
+
+let recordPrevSibling: option<Dom.node> = aRecord->previousSibling
+let recordNextSibling: option<Dom.node> = aRecord->nextSibling
+let recordAttributeName: option<string> = aRecord->attributeName
+let recordAttributeNamespace: option<string> = aRecord->attributeNamespace
+let recordOldValue: option<string> = aRecord->oldValue
```

Comment:
We should probably stick a .editorconfig on this project not sure what we want as the final line for files and maybe run a normalizer. I tried to run the rescript formatter but it changes a lot of files I think because of the line length.